pkgs/development/compilers/rust/rustc.nix

```diff
@@ -67,17 +60,17 @@ stdenv.mkDerivation {
   passthru.target = target;
 
   postPatch = ''
-    substituteInPlace src/rust-installer/gen-install-script.sh \
-      --replace /bin/echo "$(type -P echo)"
-    substituteInPlace src/rust-installer/gen-installer.sh \
-      --replace /bin/echo "$(type -P echo)"
+    # substituteInPlace src/rust-installer/gen-install-script.sh \
+    #   --replace /bin/echo "$(type -P echo)"
+    # substituteInPlace src/rust-installer/gen-installer.sh \
+    #   --replace /bin/echo "$(type -P echo)"
```

Comment:
These commented out lines in the patch command--maybe they should be deleted if not necessary?